src/test/scala/symsim/ExperimentSpec.scala

```diff
@@ -54,14 +54,21 @@ trait ExperimentSpec[State, ObservableState, Action]
    *  @param filePath The name of the file were the results are to be stored,
    *  including path and extension.
    */
+
   def evalAndLog(
     setup: concrete.ConcreteExactRL [State, ObservableState, Action],
     policies: List[setup.Policy], 
     filePath: String 
   ): Unit = 
     val rewardSamples = policies.map { setup.evaluate }
```

Comment:
This still uses just `evaluate` from ConcreteExactRL, which uses random initial states of the agent, but we wanted to control from which states we are starting. I will try to fix this.